.travis.yml

```diff
@@ -24,7 +24,7 @@ before_script:
 script:
   - yarn run test
   - yarn run lint
-  - yarn run lint:scss
+  - 'yarn run lint:scss'
```

Comment:
Please consider removing `''`

Reply:
Apparently VS Code Yaml Formatting considered it dangerous due to ':' character, is it ok? @FieryCod

Reply:
Let it be like it is if the extension thinks it's dangerous. It worked previously but ok :D Merge when you'll be ready.